graphql/schema.graphql

```diff
@@ -984,6 +820,14 @@ type Query {
 	Basic information about the currently authenticated user, if any.
 	"""
 	userInfo: UserInfo!
+	"""
+	Gets all bookmarks for the currently authenticated user.
+	"""
+	getBookmarks: [UUID!]
```

Comment:
With 2, I think leaving it as returning the document id list could make it reusable if need be. Unless there's a large disadvantage to doing so that I don't know about.